scripts/build-coredns

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+source $(dirname $0)/../scripts/lib/debug_functions
+
+function cleanup {
+    trap_commands
+    rm -rf ${TOPDIR}/coredns
+}
+
+trap cleanup EXIT
+
+COREDNS_VERSION=$5
+TOPDIR=$(git rev-parse --show-toplevel)
+COREDNS_GIT="https://github.com/coredns/coredns.git"
+DOCKER_TAG="lighthouse-coredns:dev"
+BUILD_DIR="build/linux/amd64"
+
+cd ${TOPDIR}
+git clone -b v${COREDNS_VERSION} ${COREDNS_GIT}
+cd ${TOPDIR}/coredns
+sed -i "\$alighthouse:github.com/submariner-io/lighthouse/plugin/lighthouse" plugin.cfg
+go mod vendor
+mkdir -p ${TOPDIR}/coredns/vendor/github.com/submariner-io/lighthouse/plugin/ ${BUILD_DIR}
+cp -R ${TOPDIR}/plugin ${TOPDIR}/coredns/vendor/github.com/submariner-io/lighthouse/
```

Comment:
(not sure), do you need -f too to make sure it overwrites existing old files? 

another option would be a link?

Reply:
cp -R seems to overwrite,

but I would delete the directory, and then copy again, some files could have been deleted in the new sources of lighthouse plugin, and .. if present make compilation fail.

A link would also solve it (but not sure if that's container-happy)

Reply:
The coredns folder and its vendor will be removed each run , there a trap exit that deleted the folder, this ensures every run is a new code.

Reply:
Oh righty!, I missed that part thanks :)